### PR TITLE
Add piratehash

### LINKED ISSUE #31: New entry submission - Pedro Gonçalves
New submission:

Name: Pedro Gonçalves
Url: piratehash.com
Description: Bitcoin curiosities.
Type: Merchant - Education

### LINKED ISSUE #35: New entry submission - Reckless Coffee
New submission:

Name: Reckless Coffee
Url: https://reckless.coffee
Description: Specialty grade selection of coffees from Puerto Rico. Bitcoin-only. (P2EP enabled)
Type: Merchant - Food

### PR DESCRIPTION
**Pull request by pavlenex**
closes #33
closes #35

### CODE REVIEW

File: directory/src/components/Directory/List.js [hunk 1]
```diff
@@ -1029,4 +1029,12 @@ export const list = [
     description:
       "Ramblingsales.com sells crypto apparel and uses BTCPay Server on checkout."
   },
+  {
+    name: "Pedro Gonçalves",
```
Comment (joerlop) [inline directory/src/components/Directory/List.js:1033]: I think Pedro Goncalves is the owner's name. Maybe name here should be Pirate Hash?

Comment (pavlenex) [reply directory/src/components/Directory/List.js:1033]: Yeah, that's the way submission was made so didn't want to alter. I believe people are used to email forms and associate name with a name. How about we put Project/Company name as the name of the field.

Comment (joerlop) [reply directory/src/components/Directory/List.js:1033]: Good idea. Will do.

Comment (pavlenex) [reply directory/src/components/Directory/List.js:1033]: I will now modify the store name :)

Comment (pavlenex) [reply directory/src/components/Directory/List.js]: @joerlop fixed now.

**Pull request merged**